airframe-ulid/src/main/scala/wvlet/airframe/ulid/ULID.scala

```diff
@@ -329,3 +329,7 @@ object ULID {
   }
 
 }
+
+case class ResourceID(prefix: String, ulid: ULID, delimiter: Char = ':') {
+  override def toString: String = s"$prefix$delimiter$ulid"
```

Comment:
Wrap individual variables with `${...}` for consistency with other code. Actually, always wrapping it is preferred for readability